lucene/analysis/kuromoji/src/java/org/apache/lucene/analysis/ja/dict/UserDictionary.java

```diff
@@ -104,6 +104,8 @@ public int compare(String[] left, String[] right) {
     long ord = 0;
     
     for (String[] values : featureEntries) {
+      String surface = values[0].replaceAll(" ", "");
```

Comment:
maybe replace all white space here (ie including tabs) by using \s?

Reply:
ah never mind my question, I see from the email discussion that is the case: we shouldn't allow this kind of entry

Reply:
> maybe replace all white space here (ie including tabs) by using \s?

Thanks, fixed.